keyboards/lfkeyboards/mini1800/info.json

```diff
@@ -0,0 +1,13 @@
+{
+    "keyboard_name": "Mini1800", 
+    "url": "", 
+    "maintainer": "qmk", 
+    "bootloader": "", 
+    "width": 19.5, 
+    "height": 5.25, 
+    "layouts": {
+        "DEFAULT": {
```

Comment:
This needs to match the keyboard's "macro" name. 

In this case, it appears to be `LAYOUT`.
https://github.com/devbryce/qmk_firmware/blob/371bf146b5d454acb85eec1c6b5f9817c310d70a/keyboards/lfkeyboards/mini1800/mini1800.h#L72